code/client/munkilib/munkicommon.py

```diff
@@ -1212,7 +1212,8 @@ def pref(pref_name):
         'SuppressUserNotification': False,
         'SuppressAutoInstall': False,
         'SuppressStopButtonOnInstall': False,
-        'PackageVerificationMode': 'hash'
+        'PackageVerificationMode': 'hash',
+        'FollowTTPRedirects': 'none',
```

Comment:
Well that's embarrassing. I must have sausage fingered it just before committing.